Gopkg.toml

```diff
@@ -46,12 +46,12 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/backoff"
 
-[[constraint]]
-  branch = "master"
+[[override]]
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/e2esetup"
 
-[[constraint]]
-  branch = "master"
+[[override]]
```

Comment:
Sorry, this should be `constraint` since it's a direct dependencies. If we have here e2e-harness as transitive dependency too only then it should be added as `override` too.

Reply:
@sslavic Should be good now. PTAL